### DISCUSSION TITLE
suppressing optimizer result

### DISCUSSION THREAD

**Discussion by YoonyongAhn** [do-mpc/do-mpc#146] [Q&A] [answered]
Hello.
I'm trying to implement a real-time MPC (using do-mpc) into a little vehicle robot via ROS.
I learned that it is possible to suppress the optimizer result using '%%capture' command from your examples.
But I realized that the command is working only in a jupyter notebook.
I'm wondering if I can suppress IPOPT result **only** which is coming out at each control iteration on Ubuntu terminal.

Thank you.

**Comment by 4flixt** [comment, marked as answer]
Hi @YoonyongAhn,

thats simple: Check the [docs](https://www.do-mpc.com/en/latest/api/do_mpc.controller.MPC.set_param.html) here. The **Note** on the bottom of the page holds the answer for you.

**Reply by YoonyongAhn** [reply]
It worked!
Thx a lot.
